### ISSUE TITLE
Idea for Compare Page in the Website

### ISSUE THREAD

**Issue by aditya-chari25** [MAVRICK-1/Nest-Ondc#75]
**Feature Request.**
The Compare Page is not present in the website and I have a idea. 

**Describe the solution you'd like**
1. Basically for every product we'll add the compare icon (already present) and those stuff will be added to the compare page.
2. In the compare page between the items selected we can display the items vertically since for comparison will be easier.
3. In every item only the price ,description, by whom and distance will be present (this can be present in tabular column) for easier comparison
4. And while adding we have to update the no. of comparisons above the Compare icon present in top right.

basically the vertical description can be like this below.

![example-comp](https://github.com/MAVRICK-1/Nest-Ondc/assets/84094715/b9ce9be0-c994-430c-8824-ec6ab45ef8b8)

**Describe alternatives you've considered**
1. An additional feature can be cross-topic item comparison that is, if we select items from electronics and groceries
   we can create comparisons among two different topics which are present.

@MAVRICK-1 Please assign me this issue under SSOC'24!!
**

**Comment by akbatra567** [comment]
A following issue was also created and for that PR is also merged.
https://github.com/MAVRICK-1/Nest-Ondc/issues/101
